crates/db_views/src/post_view.rs

```diff
@@ -334,10 +335,9 @@ impl<'a> PostQueryBuilder<'a> {
         .then_order_by(post_aggregates::stickied.desc());
     }
 
-    if let Some(community_name) = self.community_name {
+    if let Some(community_actor_id) = self.community_actor_id {
       query = query
-        .filter(community::name.eq(community_name))
-        .filter(community::local.eq(true))
+        .filter(community::actor_id.eq(community_actor_id))
         .then_order_by(post_aggregates::stickied.desc());
     }
```

Comment:
The strategy now instead for the post and comment query builders, is to filter by the actor id instead of the local only name, since this can query for any user.